about/index.html

```diff
@@ -131,9 +131,9 @@ <h3 class='card-title'>Chad Elish</h3>
                         Apple IT consultant for a small firm.
                       </li>
                     </ul>
-                    <p><a href="http://www.linkedin.com/in/chadelish">LinkedIn</a>&nbsp&nbsp
-                      <a href="http://www.twitter.com/chadisrad">Twitter</a>&nbsp&nbsp
-                      <a href="http://facebook.com/israd">Facebook</a></p
+                    <p><a href="http://www.linkedin.com/in/chadelish">LinkedIn</a>&nbsp;&nbsp;
```

Comment:
Could you actually replace these with spaces instead? I think it is a copy/paste issue